frontend/packages/db-structure/src/parser/schemarb/parser.ts

```diff
@@ -449,12 +449,17 @@ function extractForeignKeyTableNames(
     )
   }
 
-  const [foreignTableName, primaryTableName] = stringNodes.map(
-    (node): string => {
-      if (node instanceof StringNode) return node.unescaped.value
-      return ''
-    },
-  ) as [string, string]
+  const [firstNode, secondNode] = stringNodes.map((node): string => {
+    if (node instanceof StringNode) return node.unescaped.value
+    return ''
+  })
+  if (!firstNode || !secondNode) {
+    return err(
+      new UnexpectedTokenWarningError('Expected exactly 2 string values'),
```

Comment:
`UnexpectedTokenWarningError` is an error class that allows processing to continue with only an error noted in the log. Can you please determine if this class is really appropriate and if processing should continue or terminate?